web-api/src/persistence/postgres/messages/getMessageThreadByParentId.ts

```diff
@@ -0,0 +1,23 @@
+import { Message } from '@shared/business/entities/Message';
+import { getDbReader } from '@web-api/database';
+import { transformNullToUndefined } from '@web-api/persistence/postgres/utils/transformNullToUndefined';
+
+export const getMessageThreadByParentId = async ({
+  parentMessageId,
+}: {
+  parentMessageId: string;
+}): Promise<Message[]> => {
+  const messages = await getDbReader(reader =>
+    reader
+      .selectFrom('message')
+      .where('parentMessageId', '=', parentMessageId)
+      .selectAll()
+      .execute(),
+  );
+
+  return messages.map(result =>
```

Comment:
I do not think we should be validating messages coming from our Database, only returning raw data. So something like .

return Message(...stuff).toRawObject();